packages/@storefront/breadcrumbs/src/breadcrumbs/index.ts

```diff
@@ -33,16 +34,21 @@ class Breadcrumbs {
     switch (this.props.storeSection) {
       case StoreSections.PAST_PURCHASES:
         this.subscribe(Events.PAST_PURCHASE_SELECTED_REFINEMENTS_UPDATED, this.updateFields);
+        this.subscribe(Events.PAST_PURCHASE_NAVIGATIONS_UPDATED, this.updateFields);
+        this.subscribe(Events.PAST_PURCHASE_NAVIGATIONS_UPDATED, this.updateSelectedNavigations);
         navigationsSelector = () => this.select(Selectors.pastPurchaseNavigations);
         break;
       case StoreSections.SEARCH:
         this.subscribe(Events.ORIGINAL_QUERY_UPDATED, this.updateOriginalQuery);
         this.subscribe(Events.CORRECTED_QUERY_UPDATED, this.updateCorrectedQuery);
         this.subscribe(Events.NAVIGATIONS_UPDATED, this.updateFields);
+        this.subscribe(Events.NAVIGATIONS_UPDATED, this.updateSelectedNavigations);
         navigationsSelector = () => this.select(Selectors.navigations);
         break;
     }
-    this.state = { navigationsSelector, fields: this.getFields(navigationsSelector()), originalQuery: this.select(Selectors.query) };
+
+    // tslint:disable-next-line max-line-length
```

Comment:
At this point, I would just split the object on to separate lines.

Reply:
Will do, resolving conversation.